__init__.py

```diff
@@ -197,6 +197,28 @@ def _handle_switch(self, message):
         self.log.debug("Entity: %s" % entity)
         self.log.debug("Action: %s" % action)
 
+        # Handle turn on/off all intent
```

Comment:
if this handles both switch and light domains, should this code really live in `_handle_switch`?

it feels like "something" isn't right, if we're setting domain away from `switch` in this method

Reply:
OK, renamed